modules/authentication/server-ts/social/linkedIn/index.js

```diff
@@ -0,0 +1,37 @@
+import passport from 'passport';
+import { Strategy as LinkedInStrategy } from 'passport-linkedin-oauth2';
+import settings from '../../../../../settings';
+import AuthModule from '../AuthModule';
+
+const { clientID, clientSecret, scope, callbackURL, enabled } = settings.auth.social.linkedin;
+
+const middleware = (app, { data }) => {
```

Comment:
Lets rename `data` into `context` in modules, so that we have here `{ context }`